http_test.go

```diff
@@ -33,3 +36,22 @@ func TestHTTPHandler(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+func TestFormatRequest(t *testing.T) {
```

Comment:
It doesn't really make sense to test a function by reimplementing it. How about having a fixed output and then you test if the value returned by the function matches?

Reply:
Okay thanks for the feedback. I will reimplement the tests :+1: